avro-codegen/src/main/java/com/linkedin/avroutil1/codegen/SpecificRecordClassGenerator.java

```diff
@@ -509,7 +512,7 @@ private void populateBuilderClassBuilder(TypeSpec.Builder recordBuilder, AvroRec
         }
       }
       if (field.hasDoc()) {
-        fieldBuilder.addJavadoc(field.getDoc());
+        fieldBuilder.addJavadoc(handleJavaPoetStringLiteral(field.getDoc()));
```

Comment:
nit - the method handle() is better called escapeDollarSigns() or something. more concrete.